cell_classification/segmentation_data_prep.py

```diff
@@ -209,18 +210,44 @@ def check_input(self):
         else:
             self.selected_markers = self.selected_markers
 
+            # check if selected markers are in conversion matrix
+            if not set(self.selected_markers).issubset(set(self.conversion_matrix.columns)):
+                raise ValueError("The selected markers are not in the conversion matrix")
+
+        # NORMALIZATION DICT
         # load or construct normalization dict
         if str(self.normalization_dict_path).endswith(".json"):
             self.normalization_dict = json.load(open(self.normalization_dict_path, "r"))
+
+            # check if selected markers are in normalization dict
+            if not set(self.selected_markers).issubset(set(self.normalization_dict.keys())):
+                raise ValueError("The selected markers are not in the normalization dict")
         else:
+            # function raises a generic FileNotFoundError if selected_marker file
+            # or segmentation_fname not in data_folders
             self.normalization_dict = self.calculate_normalization_matrix(
                 self.data_folders,
                 self.selected_markers,
             )
+        # check if normalization_quantile is in [0, 1]
+        if self.normalization_quantile < 0 or self.normalization_quantile > 1:
+            raise ValueError("The normalization_quantile is not in [0, 1]")
 
+        # CELL TYPE TABLE
```

Comment:
We should check that this path is valid with [validate_path](https://github.com/angelolab/ark-analysis/blob/main/ark/utils/io_utils.py#L7)